src/xchain/modules/folks-gmp.ts

```diff
@@ -296,4 +355,13 @@ export const util = {
     );
     return await this.spokeToHubMessageFee(adapterId, startFolksChainId, undefined, receiverValue, gasLimit);
   },
+
+  async calcHubToSpokeMsgValue(msgValueEstimationArgs: MsgValueEstimationArgs): Promise<bigint> {
```

Comment:
Do you think we should be using this function to actually calculate the message fee in `retry` and `reverse`? The intention behind `util.hubToSpokeMessageFee` was to give an estimation of the message fee. Rather to get the exact fee we would need the exact message like we are doing in the other operations

Reply:
Updated using correct `messageToSend` 842694f2970aa711b6473283e007a5968f8eb416